assets/scripts/app/routing.js

```diff
@@ -54,6 +55,17 @@ export function goGoogleSignIn () {
   })
 }
 
+export function goUniversalSignIn (loginHint, screenHint) {
+  const auth0 = Authenticate()
+  auth0.authorize({
+    responseType: 'code',
+    redirectUri: AUTH0_SIGN_IN_CALLBACK_URL,
+    loginHint,
+    screenHint,
+    uiLocales: store.getState().locale.locale
```

Comment:
@louh Okay, so! I _can_ do what you said and actually access the Redux store in React-land over in `SignInDialog.jsx`. But I realized there's no reason to make it a parameter on `goUniversalSignIn()` _except_ for that reason; we _always_ want to set `uiLocales` to the state's locale and would never pass in a different value. (That I know of?)

So are there any performance or best-practice reasons you wouldn't want us to access the Redux store here? I'm very Redux-ignorant. Happy to move it over if necessary.

Reply:
This should be fine!